server/images.go

```diff
@@ -228,27 +229,26 @@ func GetModel(name string) (*Model, error) {
 			model.AdapterPaths = append(model.AdapterPaths, filename)
 		case "application/vnd.ollama.image.projector":
 			model.ProjectorPaths = append(model.ProjectorPaths, filename)
+		case "application/vnd.ollama.image.prompt":
```

Comment:
I doubt this would effect many people where this is so old, but deprecating this feels like it should be its own change rather than mixed in with this PR

Reply:
my bad I thought it was already deprecated